papers/v2024.1/paper.bib

```diff
@@ -29,6 +17,56 @@ @article{pyscf
 }
 
 
+@article{Lee2019,
+author = {Lee, Changyeon and Lee, Seungjin and Kim, Geon-U and Lee, Wonho and Kim, Bumjoon J.},
+title = {Recent Advances, Design Guidelines, and Prospects of All-Polymer Solar Cells},
+journal = {Chemical Reviews},
+volume = {119},
+number = {13},
+pages = {8028-8086},
+year = {2019},
+doi = {10.1021/acs.chemrev.9b00044},
+    note ={PMID: 31181904},
+URL = {https://doi.org/10.1021/acs.chemrev.9b00044},
+eprint = {https://doi.org/10.1021/acs.chemrev.9b00044}
+}
+
+@article{Svaneborg2023,
+    author = {Svaneborg, Carsten and Everaers, Ralf},
+    title = "{Multiscale equilibration of highly entangled isotropic model polymer melts}",
+    journal = {The Journal of Chemical Physics},
+    volume = {158},
+    number = {5},
+    pages = {054903},
+    year = {2023},
+    month = {02},
+    abstract = "{We present a computationally efficient multiscale method for preparing equilibrated, isotropic long-chain model polymer melts. As an application, we generate Kremer–Grest melts of 1000 chains with 200 entanglements and 25 000–2000 beads/chain, which cover the experimentally relevant bending rigidities up to and beyond the limit of the isotropic–nematic transition. In the first step, we employ Monte Carlo simulations of a lattice model to equilibrate the large-scale chain structure above the tube scale while ensuring a spatially homogeneous density distribution. We then use theoretical insight from a constrained mode tube model to introduce the bead degrees of freedom together with random walk conformational statistics all the way down to the Kuhn scale of the chains. This is followed by a sequence of simulations with carefully parameterized force-capped bead–spring models, which slowly introduce the local bead packing while reproducing the larger-scale chain statistics of the target Kremer–Grest system at all levels of force-capping. Finally, we can switch to the full Kremer–Grest model without perturbing the structure. The resulting chain statistics is in excellent agreement with literature results on all length scales accessible in brute-force simulations of shorter chains.}",
+    issn = {0021-9606},
+    doi = {10.1063/5.0123431},
+    url = {https://doi.org/10.1063/5.0123431},
+    eprint = {https://pubs.aip.org/aip/jcp/article-pdf/doi/10.1063/5.0123431/16706372/054903\_1\_online.pdf},
+}
+
+
+
+
+
+
+@Inbook{engl2000regularization,
```

Comment:
@DRosen285 I found a newer version of the same authors, does that work for you?

Reply:
I don't have access to the paper.
If they talk about Tikhonov regularization, it's fine.

Reply:
Confirmed, they do!